devito/passes/iet/misc.py

```diff
@@ -144,28 +144,31 @@ def generate_macros(graph, **kwargs):
 
 
 @iet_pass
-def _generate_macros(iet, tracker=None, **kwargs):
+def _generate_macros(iet, tracker=None, lang=None, **kwargs):
```

Comment:
repeating a comment I made earlier on -- it should just be part of kwargs, if not already

Reply:
As above, that would be a circulare setup with the kwargs and cls_selector